src/db.ts

```diff
@@ -1009,6 +1009,19 @@ export const saveAccountToBusinessId = async (
   ]);
 };
 
+export const getBusinessIdByAccount = async ({
```

Comment:
is it still needed?

Reply:
fixed by https://github.com/kontist/mock-solaris/pull/210/commits/b7425873de243ffe20b1c6315a4513ec5bbfb809